RELEASE_NOTES.md

```diff
@@ -27,240 +27,184 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
 
 ## The Big Ones
 
-- A full rework of the GUI Gtk/CSS. The whole GUI is now fully
-  controlled by CSS rules. There is no more size, color, position in
+- A full rework of the GUI. The whole GUI is now fully
+  controlled by [GTK+ CSS rules](https://developer.gnome.org/gtk3/stable/chap-css-overview.html). There is no more size, color, position in
   Gtk C code. This makes the whole GUI themable. This version comes
-  with different theme:
+  with several themes:
 
      - darktable                : the default theme
 
      - darktable-elegant-darker : more condensed fonts
-                                  best experience if Roboto font installed
+                                  best experience with Roboto font installed
 
      - darktable-icons-darker   : as elegant, with IOP icons
-                                  best experience if Roboto font installed
+                                  best experience with Roboto font installed
 
      - darktable-elegant-dark   : lighter version
      - darktable-elegant-grey   : even lighter version
 
-     - darktable-icons-dark     : lighter version
-     - darktable-icons-grey     : even lighter version
+     - darktable-icons-dark     : lighter version. with IOP icons
```

Comment:
Maybe **module** instead of **IOP** ?

Reply:
Yes, definitely... I think I didn't even really parse "IOP" on the first pass. I'd like to look at the whole thing one more time after some sleep, there are a couple of things I think might need a little more explanation (deprecated modules, etc).

Reply:
Ok, I guess filmic was the only actual module deprecation? I thought there had been a few more. I made the 'IOP' changes and a few minor fix-ups, should be done for now.

Reply:
Yes, filmic was the only deprecation since 2.6.3.

Reply:
Some others have been proposed in a PR, probably still open but this is for later.